### ISSUE TITLE
Translation does not work under FreeBSD 11

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#512]
This issue has been found on FreeBSD version 11.2-RELEASE-p10 (but it does not exist on tested version 12.0-RELEASE-p3) when running on develop branch of Zonemaster-Engine ([3d0c148]) and Zonemaster-Backend (a9ede9c). They have both the same versions of p5-Locale-Msgfmt (0.15_2), p5-Locale-libintl (1.31) and perl5 (5.28.1_1).

This issue is possibly related to issue zonemaster/zonemaster-engine#546.

When the test result is returned from the Backend RPC, then strings are not translated. In the extract below, the strings should have been translated to English (i.e., msgstr should be shown, not msgid), but are not. Compare with zonemaster/zonemaster-engine#546.

```
# curl -s -H "Content-Type: application/json" -d '{"jsonrpc": "2.0","id": 6,"method": "get_test_results","params": {"id": "23dc863d3490ee17","language": "en"} }' http://localhost:5000/ | jq .
{
  "jsonrpc": "2.0",
  "result": {
    "hash_id": "23dc863d3490ee17",
    "params": {
      "queue": 0,
      "domain": "zonemaster.net",
      "priority": 10,
      "ipv6": false
    },
    "id": 3,
    "creation_time": "2019-05-19 10:25:49",
    "results": [
(...)
      {
        "level": "INFO",
        "module": "ADDRESS",
        "message": "Reverse DNS entry exist for all Nameserver IP addresses.\n"
      },
      {
        "module": "ADDRESS",
        "level": "INFO",
        "message": "All reverse DNS entry matches name server name.\n"
      },
(...)
```

[3d0c148]: https://github.com/zonemaster/zonemaster-engine/commit/3d0c148d65bbab5c2a0fdca8bed76c04e3e25bfc

**Comment by mattias-p** [comment]
This is pretty much a duplicate of #353, albeit with some additional details.

**Comment by mattias-p** [comment]
Partially fixed by #526. #530 was created for the remaining problem.
